packages/core/src/components/Showcase/index.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { makeStyles, Container, Grid, Typography } from "@material-ui/core";
+
+import StoryList from "./StoryList";
+
+const useStyles = makeStyles((theme) => ({
+  showCaseContainer: {
+    maxWidth: "81.3571429rem",
+    padding: 0,
+  },
+  root: {
+    flexGrow: 1,
+    padding: "3.1875rem 1.875rem",
+    [theme.breakpoints.up("md")]: {
+      padding: "3.1875rem 0",
+    },
+  },
+  headline: {
+    width: "100%",
+    marginBottom: "2rem",
+    [theme.breakpoints.up("md")]: {
+      width: "59.625rem",
+    },
+    [theme.breakpoints.up("lg")]: {
+      width: "79.5rem",
+    },
+  },
+  headlineTitle: {
+    textAlign: "left",
+    paddingBottom: "1rem",
+  },
+  headlineDescription: {
+    textAlign: "left",
+  },
+}));
+
+function Showcase({
+                    stories,
+                    title,
+                    description,
+                    xs,
+                    cellHeight,
+                    height,
+                    minHeight,
+                    }) {
+  const classes = useStyles();
```

Comment:
```suggestion
                    minHeight,
                    ...props
                    }) {
  const classes = useStyles(props);
```

This way, we can customize via `classes` and we wouldn't need `minHeight` as prop since it's only used in styles (unlike `cellHeight` which must be a prop).